src/gui/widgets/multiline_text.hpp

```diff
@@ -208,14 +209,19 @@ class multiline_text : public text_box_base
 	/** Line number of text */
 	unsigned line_num_;
 
+	bool init_cache_;
```

Comment:
This is unused.

Reply:
ok so I'm kind of curious, why does only the macOS Release build catch this?

Reply:
fixed.